src/config/aws-post-config.ts

```diff
@@ -74,6 +79,31 @@ Without management account credentials ${CLIName} won't be able to execute the c
     writeConfig(config);
   }
 
+  private async configureAccountAccessRole(config: Config) {
+    if (config.aws.accountAccessRole) {
+      console.log(
+        `There was an AWS account access role defined: '${config.aws.accountAccessRole}'. To change this role run '${CLICommand} config aws'`,
+      );
+      return;
+    }
+
+    console.log(
+      `There is no AWS account access role selected. You must set a role with access to the account you want to scan. This role usually is called the 'OrganizationAccountAccessRole'.`,
+    );
+
+    // Prompt the user with the found profiles.
+    const selectedRole: string = await Input.prompt({
+      message:
+        "Choose an AWS CLI profile with access to the management account",
+      suggestions: ["OrganizationAccountAccessRole"],
+    });
+
+    console.log(`You have selected: ${selectedRole}`);
```

Comment:
once we have changed the `Select prompt` to `Input prompt`, i am not sure that is there any selection anymore. maybe this output can be a little bit different.

Reply:
I changed the wording to "chosen".